platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/ComboBox.kt

```diff
@@ -69,6 +67,8 @@ import org.jetbrains.jewel.ui.theme.popupContainerStyle
  * This component provides a standard dropdown UI with a text label. When clicked, it displays a popup with customizable
  * content. Supports keyboard navigation, focus management, and various visual states.
  *
+ * Note that not providing a defined size, the component will resize based on the label content.
```

Comment:
```suggestion
 * It is **strongly** recommended to provide a fixed width for the component, by
 * using modifiers such as `width`, `weight`, `fillMaxWidth`, etc. If the component
 * does not have a fixed width, it will size itself based on the label content. This
 * means the width of the component will change based on the selected item's label.
```

Same on the other KDocs :)